README.org

```diff
@@ -136,6 +136,15 @@
     removes the first part of the path when deciding where to put the
     links.
 
+** Inspiration
+The following blog posts inspired the creation of the Impermanence modules:
+
+- https://elis.nu/blog/2020/05/nixos-tmpfs-as-root/ --- [[https://github.com/etu/][@etu]]'s blog post walks
+  the reader through a NixOS-on-tmpfs installation.
+- https://grahamc.com/blog/erase-your-darlings --- [[https://github.com/grahamc/][@grahamc]]'s blog post details
+  how to utilize ZFS snapshots to clear state at boot as well as how to retain
+  various pieces of state.
```

Comment:
In my very humble opinion my post is more about _why_ and less about the _how_, but also happens to have a small amount of how at the end.

Reply:
Changed! Let me know if it can be improved further.